OpenRA.Mods.Common/Traits/ExternalCaptures.cs

```diff
@@ -34,6 +34,10 @@ class ExternalCapturesInfo : ITraitInfo
 
 		[VoiceReference] public readonly string Voice = "Action";
 
+		[GrantedConditionReference]
+		[Desc("Condition granted when capturing.")]
+		public readonly string CapturingCondition = "capturing";
```

Comment:
Shall I set a default value here ? What's the point of forcing the modder to set it ?

Reply:
No, please default this to nothing.  Forcing the modder to enable it prevents bugs when names implicitly overlap, and helps a tiny bit with performance (no point paying the cost of managing conditions that are never used).